products/datafusion/api.yaml

```diff
@@ -0,0 +1,154 @@
+async: !ruby/object:Api::Async
+  operation: !ruby/object:Api::Async::Operation
+    path: 'name'
+    base_url: '{{op_id}}'
+    wait_ms: 1000
+  result: !ruby/object:Api::Async::Result
+    path: 'response'
+    resource_inside_response: true
+  status: !ruby/object:Api::Async::Status
+    path: 'done'
+    complete: true
+    allowed:
+      - true
+      - false
+  error: !ruby/object:Api::Async::Error
+    path: 'error'
+    message: 'message'
+apis_required:
+  - !ruby/object:Api::Product::ApiReference
+    name: Cloud Data Fusion API
+    url: https://console.cloud.google.com/apis/library/datafusion.googleapis.com
+objects:
+  - !ruby/object:Api::Resource
+    name: 'Instance'
+    base_url: "projects/{{project}}/locations/{{region}}/instances"
+    create_url: projects/{{project}}/locations/{{region}}/instances?instanceId={{name}}
+    update_verb: :PATCH
+    min_version: beta
+    description: |
+      Represents a Data Fusion instance.
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation':
+          'https://cloud.google.com/data-fusion/docs/'
+      api: 'https://cloud.google.com/data-fusion/docs/reference/rest/v1beta1/projects.locations.instances'
+    parameters:
+      - !ruby/object:Api::Type::String
+        name: 'region'
+        url_param_only: true
+        input: true
+        description: |
+          The region of the Data Fusion instance.      
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'name'
+        required: true
+        input: true
+        description: |
+          The ID of the instance or a fully qualified identifier for the instance. 
+      - !ruby/object:Api::Type::String
+        name: 'description'
+        description: |
+         An optional description of the instance.
+        input: true
+      - !ruby/object:Api::Type::Enum
+        name: 'type'
+        required: true
+        description: |
+          Represents the type of Data Fusion instance. Each type is configured with 
+          the default settings for processing and memory.
+          - BASIC: Basic Data Fusion instance. In Basic type, the user will be able to create data pipelines 
+          using point and click UI. However, there are certain limitations, such as fewer number 
+          of concurrent pipelines, no support for streaming pipelines, etc.
+          - ENTERPRISE: Enterprise Data Fusion instance. In Enterprise type, the user will have more features 
+          available, such as support for streaming pipelines, higher number of concurrent pipelines, etc.
+        values:
+          - :BASIC
+          - :ENTERPRISE
+        input: true
+      - !ruby/object:Api::Type::Boolean
+        name: 'enableStackdriverLogging'
+        description: |
+          Option to enable Stackdriver Logging.
+        input: true
+      - !ruby/object:Api::Type::Boolean
+        name: 'enableStackdriverMonitoring'
```

Comment:
This can't be updated on an existing instance?

Reply:
I didn't see an option for it on the console. But just tried it out, and the flags can totally be updated on an existing.